nativeauthenticator/nativeauthenticator.py

```diff
@@ -25,13 +41,40 @@ def add_new_table(self):
             User.info = relationship(UserInfo, backref='users')
             UserInfo.__table__.create(self.db.bind)
 
+    def exceed_atempts_of_login(self, username):
```

Comment:
This method does two things:

1. Check if the user has exceeded their limit
2. Record that the user has attempted a log in

Instead, this should be three functions:

1. Records that the user has attempted to log in (mutating the dictionary)
2. Check if the user has exceeded their limit (check the dictionary)
3. Reset user's entry in dictionary after a successful login

This clearly separates different actions on the data, making it easier to understand in the longer run.